optaweb-employee-rostering-backend/pom.xml

```diff
@@ -333,6 +285,20 @@
         </plugins>
       </build>
     </profile>
+    <profile>
```

Comment:
One last thing. This profile should go to the parent as well so that the build fails if formatting is broken in benchmark or any other module.